packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -217,23 +218,28 @@ export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends
 	 * @param revision - The revision to associate with the change.
 	 * Defaults to a new, randomly generated, revision if not provided.
 	 */
-	protected applyChange(change: TChange, revision?: RevisionTag): void {
-		const commit = {
+	protected applyChange(change: TChange, revision?: StableId): void {
```

Comment:
Why did we change the type from `RevisionTag` to `StableId`?

Reply:
it's more stable